addons/binding/pom.xml

```diff
@@ -37,6 +37,7 @@
     <module>org.openhab.binding.squeezebox</module>
     <module>org.openhab.binding.tesla</module>
     <module>org.openhab.binding.vitotronic</module>
+    <module>org.openhab.binding.harmonyhub</module>
     <module>org.openhab.binding.zwave</module>
   </modules>
 
```

Comment:
Could you please also add your binding in https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml?